glotaran/builtin/io/yml/yml.py

```diff
@@ -48,6 +48,13 @@ def load_model(self, file_name: str) -> Model:
 
         spec = sanitize_yaml(spec)
 
+        if "include" in spec:
+            includes = spec.pop("include")
+            if not isinstance(includes, list):
+                includes = [includes]
+            for include in includes:
+                spec.update(self._load_yml(include))
```

Comment:
We should check if there are duplicated entries and at least warn users if they are going to be overwritten.
Atm, we just overwrite things by order, and even more dangerous we overwrite at the top level! 
Image the following scenario:
A user defines each megacomplex in its own file and includes both, all but the last one get overwritten without any warning.

```python
In [1]: spec = {"megacomplex": {"m1": {"foo":1}}}

In [2]: mc_include = {"megacomplex": {"m2": {"bar":2}}}

In [3]: spec.update(mc_include)

In [4]: spec
Out[4]: {'megacomplex': {'m2': {'bar': 2}}}
```